### ISSUE TITLE
Fail to run the experiment

### ISSUE THREAD

**Issue by KYuuto1006** [ReneeYe/ConST#4]
Hi, when I was using `bash ConST/scripts/train_en2x.sh de checkpoint/model_saved.` to train the model, I encountered some problem. It would be great help to me if you can help taking a look at my bug.
In the original ConST/scripts/train_en2x.sh file, language prefix token looks like this:
![image](https://user-images.githubusercontent.com/102947073/188795552-a48f2c6b-6296-409a-bb59-76d458524087.png)
which brings me the following error:
![image](https://user-images.githubusercontent.com/102947073/188795902-e0686df4-a62b-4ee1-9beb-4f8a400c283a.png)
After deleting the '<' and '>' outside `lang:${TGT_LANG}`, it can start training on _train_st_ but when it began to validate on _dev_st_, another assertion error occured like this:
![image](https://user-images.githubusercontent.com/102947073/188796874-55925ca3-147f-4f52-bf61-31ddf0fbbae9.png)
which comes from _fairseq/tasks/speech_to_text_triplet_with_extra_mt.py_, line 377.
I tried adding  '<' and '>' in _speech_to_text_triplet_with_extra_mt.py_ before assertion, but it didn't work. It would be of great help to me if you know some solutions. Thank you!

**Comment by ReneeYe** [comment]
@KYuuto1006 Try this: --lang-prefix-tok='<lang:${TGT_LANG}>', e.g. --lang-prefix-tok='<lang:fr>'

**Comment by ReneeYe** [comment]
@KYuuto1006 Do not delete '<' and '>', it is part of the language tag.😂

**Comment by KYuuto1006** [comment]
@ReneeYe Thank you very much for your reply!
Unfortunately, I have tried `--lang-prefix-tok='<lang:${TGT_LANG}>'` and `--lang-prefix-tok='lang:${TGT_LANG}'`, both of them still brought the same Assertion Error.

**Comment by ReneeYe** [comment]
@KYuuto1006 I'm pretty sure `--lang-prefix-tok='<lang:${TGT_LANG}>'` is correct. Do you have <lang:${TGT_LANG}> in your vocabulary?

**Comment by ReneeYe** [comment]
@KYuuto1006  I'm pretty sure `--lang-prefix-tok='<lang:${TGT_LANG}>'`  is correct. Do you have `<lang:${TGT_LANG}>`  in your vocabulary?

**Comment by KYuuto1006** [comment]
@ReneeYe I think it's the problem of my fairseq, I already fixed it. Thank you a lot!

**Comment by PRASHANTGB** [comment]
@KYuuto1006 ..kindly tell me how you fixed  ConST/scripts/train_en2x.sh: line 14: lang:de: No such file or directory

**Comment by KYuuto1006** [comment]
> @KYuuto1006 ..kindly tell me how you fixed ConST/scripts/train_en2x.sh: line 14: lang🇩🇪 No such file or directory
@PRASHANTGB 
I think the original code is accurate and I failed to run it due to incompatible fairseq version. But I made it work in my version by
![image](https://user-images.githubusercontent.com/102947073/212064776-a3364e6d-8e03-4e26-a850-1a3335ab117e.png)
Hope it helps. I believe the author's code is correct, we just met some environment problems.
